docs/optimize-sensors.md

```diff
@@ -17,6 +17,8 @@ Note that cached data doesn’t necessarily come from the previous analysis of t
 | Subsequent Analysis of a branch | Analyzed  branch    |
 | Pull Request | Target branch if exists, otherwise main branch |
 
+It's the plugin's responsability to invalidate cached data. Plugins shouldn't rely on file statuses (`InputFile.status()`) to conclude whether then can use cached data for that file. The reason is that in some situations, the computation of the file statuses is based on the differences compared to the same branch from where the cached data is downloaded from.
```

Comment:
responsability -> responsibility ?
whether then -> whether they

Not sure I understand the last sentence, maybe it's just because I am not familiar with the cache topic. Also the comment on the `Status` interface says: 
> Status regarding previous analysis

Should we add "on the same branch" to make it more precise ?

Reply:
Sorry for the very late reply.
I fixed the typos and I tried to clarify the last sentence.

The point is precisely that the status is not necessarily relative to the same branch.